Content.Shared/_Impstation/Clothing/Obvious/WearerGetsExamineTextSystem.cs

```diff
@@ -0,0 +1,113 @@
+            isCorrectSlot ? entity.Comp.PrefixExamineOnWearer : "obvious-prefix-have"); // uses a different prefix if worn / displayed
+    }
+
+    private void OnUnequipped(Entity<WearerGetsExamineTextComponent> entity, ref GotUnequippedEvent args)
+    {
+        //REMOVE THE FUCKIN INSPECT TEXT
+        if (entity.Comp.Wearer == null)
+            return;
+
+        var wearer = entity.Comp.Wearer.Value;
+        if (TryComp(wearer, out ExtraExamineTextComponent? obviousExamine))
+        {
+            obviousExamine.Lines.Remove(entity.Comp.ExamineOnWearer);
+        }
+
+        entity.Comp.Wearer = null;
+        Dirty(entity);
+    }
+
+    private void OnExamine(Entity<WearerGetsExamineTextComponent> entity, ref ExaminedEvent args)
+    {
+        var currentlyWorn = entity.Comp.Wearer != null;
+        var contraString = "";
+        var outString = "";
+        outString = Loc.GetString(currentlyWorn ? "obvious-on-item-currently" : "obvious-on-item",
+            ("used", Loc.GetString(entity.Comp.PocketEvident ? "obvious-type-pockets" : "obvious-type-default")),
+            ("thing", entity.Comp.Thing),
+            ("me", Identity.Entity(entity, EntityManager)));
+        if (entity.Comp.WarnExamine && !currentlyWorn && TryComp(entity, out ContrabandComponent? contra))
+        {
+            switch (contra.Severity) // apply additional text if the item is contraband to note that displaying it might be really bad
+            {
+                case "Syndicate":
+                    contraString = Loc.GetString("obvious-on-item-contra-syndicate");
+                    break;
+                case "Magical":
+                    contraString = Loc.GetString("obvious-on-item-contra-magical");
+                    break;
+                case "Major":
+                    contraString = Loc.GetString("obvious-on-item-contra-major");
+                    break;
+                default:
+                    break;
+            }
+            outString += " " + contraString;
+        }
+
+        if (entity.Comp.WarnExamine)
+        {
+            var affecting = currentlyWorn ? entity.Comp.Wearer.GetValueOrDefault() : args.Examiner;
+
+            var prefix = Loc.GetString(entity.Comp.PocketEvident ? "obvious-prefix-have" : entity.Comp.PrefixExamineOnWearer, // this is hacky but it's less noticeable than having no check here
+                ("user", Identity.Entity(affecting, EntityManager)),
+                ("name", Identity.Name(affecting, EntityManager)));
+            var examine = Loc.GetString(entity.Comp.ExamineOnWearer,
+                ("user", Identity.Entity(affecting, EntityManager)),
+                ("name", Identity.Name(affecting, EntityManager)));
+
+            var warnString = Loc.GetString("obvious-on-item-for-others",
+                ("will", currentlyWorn ? "can" : "will"),
+                ("output", prefix + " " + examine));
+
+            outString += "\n" + warnString;
```

Comment:
this works fine, for any future prs like this it might be worth looking into `StringBuilder`. i won't ask you to redo all this just to do that, though